package.json

```diff
@@ -1,68 +1,83 @@
   "main": "dist/index.js",
   "scripts": {
     "check-coverage": "istanbul check-coverage --statements 100 --branches 100 --functions 100 --lines 100",
     "report-coverage": "cat ./coverage/lcov.info | codecov",
     "commit": "git-cz",
-    "test": "mocha tests/**/*.js --compilers js:babel-core/register",
-    "test:ci": "babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/_mocha -- tests/**/*.js",
+    "test": "cross-env NODE_ENV=test babel-node node_modules/.bin/tape tests/**/*.spec.js | tap-spec",
+    "test:ci": " npm run clean && babel-node node_modules/.bin/babel-istanbul cover node_modules/.bin/tape -- tests/**/*.js",
+    "clean": "rimraf dist/* coverage",
     "test:watch": "watch 'npm test' src/ tests/",
-    "build": "babel src --out-dir dist",
-    "lint": "eslint src tests",
+    "build": "npm run clean && babel src --out-dir dist",
+    "lint": "./scripts/lint src tests",
     "semantic-release": "semantic-release pre && npm run build && npm publish && semantic-release post"
   },
   "repository": {
     "type": "git",
     "url": "https://github.com/sphereio/sphere-product-type-import.git"
   },
+  "bugs": {
+    "url": "https://github.com/sphereio/sphere-product-type-import/issues"
+  },
   "keywords": [
     "ctp",
     "commercetools",
     "product-type",
     "import"
   ],
+  "files": [
+    "dist/"
+  ],
   "author": "Philipp Sporrer <philipp.sporrer@commercetools.de>",
+  "homepage": "https://github.com/sphereio/sphere-product-type-import#readme",
   "license": "MIT",
   "devDependencies": {
-    "babel-cli": "6.6.0",
-    "babel-core": "6.6.5",
-    "babel-eslint": "5.0.0",
-    "babel-istanbul": "0.6.1",
-    "babel-preset-es2015": "6.6.0",
-    "babel-preset-stage-0": "6.5.0",
-    "babel-plugin-transform-runtime": "6.12.0",
-    "babel-runtime": "6.9.2",
-    "chai": "3.5.0",
-    "codecov.io": "0.1.6",
-    "commitizen": "2.6.1",
-    "cuid": "1.3.8",
-    "cz-conventional-changelog": "1.1.5",
-    "eslint": "2.2.0",
-    "ghooks": "1.0.3",
-    "isparta": "4.0.0",
-    "istanbul": "0.4.2",
-    "mocha": "2.4.5",
+    "babel-cli": "^6.6.0",
+    "babel-core": "^6.6.5",
+    "babel-eslint": "^7.0.0",
+    "babel-istanbul": "^0.6.1",
+    "babel-plugin-transform-runtime": "^6.12.0",
+    "babel-preset-es2015": "^6.14.0",
+    "babel-register": "^6.14.0",
+    "babel-runtime": "^6.9.2",
+    "chai": "^3.5.0",
+    "codecov": "^1.0.1",
+    "commitizen": "^2.6.1",
+    "cross-env": "^2.0.1",
+    "cuid": "^1.3.8",
+    "cz-conventional-changelog": "^1.1.5",
+    "eslint": "^3.6.1",
+    "eslint-config-airbnb": "^12.0.0",
+    "eslint-config-commercetools": "^4.0.0",
+    "eslint-formatter-pretty": "^1.0.0",
+    "eslint-plugin-import": "^1.16.0",
+    "eslint-plugin-jsx-a11y": "^2.2.2",
+    "eslint-plugin-react": "^6.3.0",
+    "ghooks": "^1.0.3",
+    "isparta": "^4.0.0",
+    "istanbul": "^0.4.2",
+    "mocha": "^2.4.5",
```

Comment:
@adnasa Also, don't forget to remove `isparta` and `istanbul` in favor of `nyc`.